### ISSUE TITLE
Theme

### ISSUE THREAD

**Issue by foXaCe** [yllibed/Zigbee2MqttAssistant#88]
hello it possible to add black theme ?

**Comment by carldebilly** [comment]
@all-contributors please add foXaCe to ideas

**Comment by ciotlosm** [comment]
I haven't tried it yet, but isn't the theme from home-assistant properly propagated?

**Comment by carldebilly** [comment]
Nope. But it's possible to use the [`prefers-color-scheme`](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-color-scheme) from the browser to know the user/OS preferences.
